packages/strapi-admin/admin/src/components/EventInput/tests/index.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { IntlProvider } from 'react-intl';
+
+import EventInput from '../index';
+
+describe('<EventInput />', () => {
```

Comment:
It this component as well it would be great to have a bit more test. Maybe you could start by making the `formatValue` a util function so you can test it more easily.